CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to Django-drip-campaigns
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
```

Comment:
I would here to make sure that travis passes

Reply:
that is stated in step 4

Reply:
Yes but you can understand that like 'local tests passes' but travis uses several versions of python and django. I would put something like `Ensure that local and travis tests pass`.
WDYT?

Reply:
Travis would run as part of the CI. and we will request changes for that anyway.
For me it's clear as is.

Reply:
OK if you think is clear let's leave it that way